src/components/AnnotationPopover/AnnotationPopover.scss

```diff
@@ -171,6 +171,8 @@
     @media #{$tablet} {
         .ba-popover {
             border-left: 1px solid $seesee;
+            // Max width matches the max-width of the Preview Sidebar
+            max-width: 341px;
```

Comment:
This doesn't seem like a desirable thing to hardcode. It already has a magic number of 45% width.

Reply:
This value is hard-coded in elements as well https://github.com/box/box-ui-elements/blob/master/src/components/ContentSidebar/SidebarContent.scss#L7
I can actually remove the 45% width w/ the hard-coded value